week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -4,8 +4,12 @@
   The business is breaking out into a new market and need to convert prices to USD
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
-
-function convertToUSD() {}
+function formatCurrency(price) {
+  return price.toFixed(2);
+}
+function convertToUSD() {
+  return price * 1.4;
+}
```

Comment:
yep that is good. I think it might be wise here to put exchange rate as a var - exchange rates change. In fact it would be good as a parameter, most likely. But what you have fulfils the exercise.